alws/crud/release.py

```diff
@@ -127,6 +128,19 @@ def get_pulp_based_response():
             'modules': rpm_modules,
         }
 
+    if not settings.package_beholder_enabled:
+        return get_pulp_based_response()
+
+    clean_base_dist_name = re.search(
+        r'(?P<dist_name>[a-z]+)', base_dist_name,
+        re.IGNORECASE).groupdict().get('dist_name')
+    clean_base_dist_name_lower = clean_base_dist_name.lower()
```

Comment:
You'll get exception if `clean_base_dist_name` is None

Reply:
Should not be, but you're right